### DISCUSSION TITLE
allow date entry without separators

### DISCUSSION THREAD

**Discussion by kurti500** [jonaswinkler/paperless-ng#926] [Feature Requests]
As the prepopulated date on my files is often wrong in paperless, I would like to suggest to allow date entry on the edit page without separators like I know it from ERP systems.
Depending on the locale e.g.  251221 would translate into 25.12.2021 or 042321 would get 04/23/2021.
This would speed up data/date entry. 
Maybe a small(?) enhancement to increase usability?
Thanks for considering.

**Comment by jonaswinkler** [comment]
How would you translate 1112021 into a date? Will that become 1.11.2021 or 11.1.2021? Separators serve a purpose, after all.

**Comment by kurti500** [comment]
Well, its just a suggestion, of course certain rules need to be followed/applied, e.g. as in my examples only 6 digit entries without separators allowed to avoid issues as you have shown.
This approach is quite common in business applications, also Excel supports this approach with some scripting, but of course I will also survive in case paperless will not support it.
It would just speed up date entry.

**Reply by shamoon** [reply]
Yea this was the idea behind the masking in #177 but ultimately that was cut for simplicity sake

**Reply by kurti500** [reply]
I see, if this was already considered and dropped, I assume this one can/needs to be closed.
